packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.tsx

```diff
@@ -7,7 +7,7 @@ import { WalletText } from '../WalletText';
 import './ModalStepWrapper.scss';
 
 type TModalStepWrapperProps = {
-    closeOnEscape?: boolean;
+    disableOnEscapeClick?: boolean;
```

Comment:
```suggestion
    disableCloseOnEscape?: boolean;
```
This naming is clearer on the functionality

Reply:
Prefix with `should` or `is`?